### PR TITLE
mysql: saner sysctl values for galera nodes

### PR DESCRIPTION
**Pull request by Itxaka**
Adds sysctl values for swappiness obtained from the tuned application
profiles for virtual-hosts in order to have less aggresive swap
in galera nodes

### CODE REVIEW

**Comment by Itxaka** [comment]
[Saner sysctl defaults for compute nodes and database nodes](https://trello.com/c/hSbEqGyx/336-saner-sysctl-defaults-for-compute-nodes-and-database-nodes)

File: chef/cookbooks/mysql/recipes/ha_galera.rb [hunk 1]
```diff
@@ -364,3 +364,22 @@
...
+template "galera_sysctl.conf" do
+  path "/etc/sysctl.d/90-galera_sysctl.conf"
+  mode "0644"
+  variables(
+    swappiness: node[:mysql][:sysctl][:swappiness]
+  )
+end
+
+bash "reload galera_sysctl.conf" do
+  code "/sbin/sysctl -e -q -p /etc/sysctl.d/90-galera_sysctl.conf"
```
Comment (nicolasbock) [inline chef/cookbooks/mysql/recipes/ha_galera.rb]: Why do we need the `-e` / `--ignore` option here? Isn't that just hiding a potential bug in the barclamp?

Comment (Itxaka) [reply chef/cookbooks/mysql/recipes/ha_galera.rb]: Its used on other parts of the cookbooks  when applying sysctl changes. Im guessing it had an historic reason for it, which I cannot find anywhere, so I opted to keep it the same for consistency reasons :thinking:

**Comment by dirkmueller** [comment]
imho this should be moved to provisioner or some other barclamp in core to ensure that it is being rolled out everywhere. 

potentially we should just use tuned instead of reinvinting wheels (I know, our wheel is going to be rounder)

**Comment by dirkmueller** [comment]
move this to provisioner-base ?

**Comment by Itxaka** [comment]
wow sorry, I do not get notifications regarding this so I thougth it was just lingering around :o @dirkmueller

**Pull request closed without merging**